WordPress/Sniffs/WhiteSpace/ObjectOperatorSpacingSniff.php

```diff
@@ -9,8 +9,8 @@
 
 namespace WordPressCS\WordPress\Sniffs\WhiteSpace;
 
-use PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\ObjectOperatorSpacingSniff as Squiz_ObjectOperatorSpacingSniff;
 use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\ObjectOperatorSpacingSniff as Squiz_ObjectOperatorSpacingSniff;
```

Comment:
How come this alias is named with `_`? Wouldn't `SquizObjectOperatorSpacingSniff` be more consistent with the naming of the classes?

Reply:
This PR isn't changing the alias, just moving the statement.

Other than that, the alias is done in a similar way as aliases used elsewhere in WPCS, though I agree, they could use more consistency:
* `Sniff` class: `use PHP_CodeSniffer\Sniffs\Sniff as PHPCS_Sniff;`
* `ValidVariableNameSniff`: `use PHP_CodeSniffer\Sniffs\AbstractVariableSniff as PHPCS_AbstractVariableSniff;` (looks like this sniff can probably be removed)
* `ObjectOperatorSpacingSniff`: `use PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\ObjectOperatorSpacingSniff as Squiz_ObjectOperatorSpacingSniff;`
* `OperatorSpacingSniff`: `use PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\OperatorSpacingSniff as PHPCS_Squiz_OperatorSpacingSniff;`

Reply:
I think we can change alias names later on, that shouldn't be a BC break, no?

Reply:
Correct, aliases (just like `use` statements) are file specific, so have no BC implications.